test/plural_resolver_test.dart

```diff
@@ -0,0 +1,62 @@
+import 'package:flutter_test/flutter_test.dart';
+import 'package:i18next/i18next.dart';
+import 'package:i18next/src/plural_resolver.dart';
+import 'package:mockito/mockito.dart';
+
+abstract class FakeCheckerKeyFunction {
+  bool call(String pluralModifier);
+}
+
+class MockCheckerKeyFunction extends Mock implements FakeCheckerKeyFunction {}
+
+void main() {
+  group('PluralResolver', () {
+    MockCheckerKeyFunction checkerKeyFnc;
+    PluralResolver subject;
+
+    setUpAll(() {
+      checkerKeyFnc = MockCheckerKeyFunction();
+      subject = PluralResolver();
+    });
+
+    test('when count is equal 1 returns empty string', () {
```

Comment:
We're using a more BDD style approach for the tests

Unless the expected results are ambigous, you can omit the result on the test name.

`"when X" -> expect(X, ...)`
`"given Y" -> expect(function(Y), ...)`

The expectations should be clear (this helps keeping the tests as minimal as possible)